operator/pkg/helm/helm.go

```diff
@@ -25,14 +25,17 @@ import (
 	"sort"
 	"strings"
 
-	"k8s.io/helm/pkg/chartutil"
-	"k8s.io/helm/pkg/engine"
-	"k8s.io/helm/pkg/proto/hapi/chart"
-	"k8s.io/helm/pkg/timeconv"
+	"helm.sh/helm/v3/pkg/chart"
+	"helm.sh/helm/v3/pkg/chart/loader"
+	"helm.sh/helm/v3/pkg/engine"
+	"sigs.k8s.io/yaml"
+
+	"helm.sh/helm/v3/pkg/chartutil"
+
+	"istio.io/pkg/log"
```

Comment:
format

Reply:
I am not sure what you are looking for here? This is run through `make format`

Reply:
make format doesn't merge blocks separated by blank lines - just remove the two blank lines and reformat

Reply:
I'll update but I think if we want to enforce style issues like this they should be encoded into `make format`/`make lint`

Reply:
Enforcement >>>> Code review convention